### PR TITLE
Allow saving onto PublicLab.org

### LINKED ISSUE #1332: Allow saving onto PublicLab.org
PublicLab.org now allows (if you're logged in) for opening an image into the editor at https://publiclab.org/post (again, must be logged in)

The docs for this are here: https://github.com/publiclab/plots2/issues/6594#issuecomment-562389677

And I've just completed this in Infragram, here:

https://github.com/publiclab/infragram/commit/7ca92dbf9f630f6010847c26f770cbada7e5a454

So, we should add one more type to the ImageSequencer "Save" box, which is "Save to PublicLab.org" -- shown here:

image

Once you do this, you should see the https://PublicLab.org/post editor open, like this:

image

You can test this out by pasting the following code into the console in https://sequencer.publiclab.org --

```js
function postToPL(imgSrc) {
  var uniq = Date.now();
  $('body').append('<form method="post" id="postToPL' + uniq + '" action="https://stable.publiclab.org/post" target="postToPLWindow"><input type="hidden" name="datauri_main_image" /></form>');
  f = $('#postToPL' + uniq)[0];
  f.datauri_main_image.value = imgSrc;
  window.open('', 'postToPLWindow');
  f.submit();
}
postToPL($('img')[0].src)
```

### PR DESCRIPTION
**Pull request by keshav234156**
Fixes #1332 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] ask `@publiclab/is-reviewers` for help, in a comment below
* [ ] Insert-step functionality is working correct as expected.
> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
Please make sure to get at least two reviews before asking for merging the PR as that would make the PR more reliable on our part
Thanks!
Untitled_ Dec 7 2019 5_38 PM

### CODE REVIEW

**Comment by keshav234156** [comment]
@jywarren Is this what you were expecting?

File: examples/demo.js [hunk 1]
```diff
@@ -116,6 +116,9 @@ window.onload = function () {
     else if (dropDownValue == 'save-seq') {
       saveSequence();
     }
+    else if (dropDownValue == 'save-to-publiclab.org' ){
+      savetopubliclab();
+    }
```
Comment (symt) [inline examples/demo.js]: Two things here: 
1. The condition of the if statement has a linting issue. It should be `else if (dropDownValue == 'save-to-publiclab.org') {`.
2. The function name should be in camel case: saveToPublicLab

File: examples/demo.js [hunk 2]
```diff
@@ -211,6 +214,18 @@ window.onload = function () {
     download(image, 'index.gif', 'image/gif');// downloadjs library function
   }
 
+  function savetopubliclab(){
```
Comment (symt) [inline examples/demo.js]: See above: The name should be camel case. There should also be a space between the () and {

File: examples/demo.js [hunk 3]
```diff
@@ -211,6 +214,18 @@ window.onload = function () {
...
+    function postToPL(imgSrc) {
+      var uniq = Date.now();
+      $('body').append('<form method="post" id="postToPL' + uniq + '" action="https://publiclab.org/post" target="postToPLWindow"><input type="hidden" name="datauri_main_image" /></form>');
+      f = $('#postToPL' + uniq)[0];
+      f.datauri_main_image.value = imgSrc;
+      window.open('', 'postToPLWindow');
+      f.submit();
+    }
+    postToPL($('img')[sequencer.steps.length - 1].src);
+  }
```
Comment (symt) [inline examples/demo.js]: Is there any benefit to nesting this function? You could do something like the following to remove the extra function call and the local variable:
```
function saveToPublicLab() {
    var uniq = Date.now();
    $('body').append('<form method="post" id="postToPL' + uniq + '" action="https://publiclab.org/post" target="postToPLWindow"><input type="hidden" name="datauri_main_image" /></form>');
    f = $('#postToPL' + uniq)[0];
    f.datauri_main_image.value = $('img')[sequencer.steps.length - 1].src;
    window.open('', 'postToPLWindow');
    f.submit();
}
```

**Comment by keshav234156** [comment]
@jywarren can you please give  your views on this as well.

**Pull request closed without merging**